Gemfile

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) { |repo_name| "https://github.com/CardTapp/acts-as-taggable-on-mongoid" }
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `repo_name`. You can omit the argument if you don't care about it.